src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -47,7 +66,11 @@ export default Utils.withCancellationSignal(class LaunchAnalysisModal extends Co
             this.doLaunch()
           }
         }, ['Launch']) :
-        h(ButtonPrimary, { onClick: onDismiss }, ['OK'])
+        h(ButtonPrimary, { onClick: onDismiss }, ['OK']),
+      suffixContent: [div({ style: { fontSize: 10 } }, [
+        sup(['*']),
+        'Note: External actions (or workflow tasks) might act to move or copy this data. It is not guaranteed to remain only in this bucket. Metadata produced by this run is stored in the US.'
+      ])]
```

Comment:
Based on my experience working with our end-users, people will fall into three categories:

80% people who don't know they can do this
15% people who are aware this is a possibility but never do it
5% people who actually write data somewhere custom

For the 80% who don't know what this means, it kind of sounds like Terra is failing to protect their data from "external actions" that could move files somewhere else at any time ("not guaranteed to remain only in this bucket").

```suggestion
```

Reply:
Have re-worded. Let me know if the new form rings truer.